benchmarks/src/main/scala/zio/redis/RedisClients.scala

```diff
@@ -0,0 +1,66 @@
+package zio.redis
+
+import cats.effect.{ Blocker, Resource, IO => CatsIO }
+import dev.profunktor.redis4cats.RedisCommands
+import dev.profunktor.redis4cats.data.RedisCodec
+import fs2.io.tcp.SocketGroup
+import io.chrisdavenport.rediculous.{ Redis, RedisConnection }
+import io.lettuce.core.ClientOptions
+import laserdisc.fs2.RedisClient
+import zio.redis.BenchmarkRuntime.{ RedisHost, RedisPort }
+
+object RedisClients {
```

Comment:
Perhaps convert these helpers to traits and just mix them in BenchmarkRuntime?

Reply:
It's good idea. Please take a look again, it looks way cleaner.

Reply:
I faced two issue after refactoring:

1. I had an exception when I tried to run jmh tests. It looks like it cannot manage generated class names properly:
```
Writing out Java source to /Users/kostjas/projects/zio-redis/benchmarks/target/scala-2.12/src_managed/jmh and resources to /Users/kostjas/projects/zio-redis/benchmarks/target/scala-2.12/resource_managed/jmh| => redis / Compile / packageBin 0s
Annotation generator had thrown the exception.
java.lang.InternalError: Malformed class name
	at java.lang.Class.getSimpleName(Class.java:1330)
	at java.lang.Class.getCanonicalName(Class.java:1399)
	at org.openjdk.jmh.generators.reflection.RFClassInfo.getQualifiedName(RFClassInfo.java:67)
```

2. Second exception is related to wrong initialization of EcecutionContext:
```
 java.lang.NullPointerException
[error] 	at cats.effect.internals.IOForkedStart$.apply(IOForkedStart.scala:47)
[error] 	at cats.effect.internals.IOStart$.$anonfun$apply$1(IOStart.scala:40)
[error] 	at cats.effect.internals.IOStart$.$anonfun$apply$1$adapted(IOStart.scala:29)
[error] 	at cats.effect.internals.IORunLoop$RestartCallback.start(IORunLoop.scala:439)
```

As I quick fix I decided to move all instances of QueryUnsafeRunner to RedisClients from companion object.
Also EffectContexts was deleted and `implicit val cs: ContextShift[CatsIO]` with `implicit val timer: Timer[CatsIO]` were placed as part of RedisClients trait